virttest/utils_test/qemu/__init__.py

```diff
@@ -613,7 +613,8 @@ def hotplug_memory(self, vm, name):
         :param vm: VM object
         :param name: memory device name
         """
-        devices = vm.devices.memory_define_by_params(self.params, name)
+        params = self.params.object_params(name)
+        devices = vm.devices.memory_define_by_params(params, name)
```

Comment:
Why not wrap line 616 into `devices.memory_define_by_params()` directly?

Reply:
@luckyh Done, wrapped it and updated relevant code.  Please help review again, thanks!